import-scripts/dmp-import-vars-functions.sh

```diff
@@ -281,30 +281,6 @@ function import_access_ddp_to_redcap {
     return $return_value
 }
 
-# Function for importing raindance cvr files to redcap
-function import_raindance_cvr_to_redcap {
```

Comment:
we probably can leave these in place .. in case we want to use them again (unless we are sure we won't be importing to raindance again)

Reply:
@sheridancbio this time it's definitely not going to be enabled again - we've finalized the set of raindance ids and confirmed our set matches the DMP-provided set of samples so nothing is missing anymore 🎉